x-pack/legacy/plugins/maps/public/components/_tooltip_selector.scss

```
@@ -0,0 +1,59 @@
+.mapTooltipSelector__propertyRow {
+  display: flex;
+
+  padding: $euiSizeXS 0;
+  border-bottom: 1px solid $euiColorLightShade;
+  background-color: $euiColorEmptyShade;
+
+  &:hover,
+  &:focus,
+  &:focus-within {
+    .mapTooltipSelector__propertyIcons {
+      display: block;
+      animation: mapPropertyIconsBecomeVisible $euiAnimSpeedFast $euiAnimSlightResistance;
+    }
+  }
+
+  .mapTooltipSelector__propertyIcons {
+    &:hover,
+    &:focus {
+      display: block;
+      animation: mapPropertyIconsBecomeVisible $euiAnimSpeedFast $euiAnimSlightResistance;
+    }
+  }
+}
+
+.mapTooltipSelector__propertyRow-isDragging {
+  @include euiBottomShadowMedium;
+}
+
+.mapTooltipSelector__propertyRow-isDraggingOver {
+  // Don't allow interaction events while layer is being re-ordered
+  // sass-lint:disable-block no-important
+  pointer-events: none !important;
+}
+
+.mapTooltipSelector__propertyContent {
+  overflow: hidden;
+  flex-grow: 1;
+  min-height: 24px;
```

```suggestion
  min-height: $euiSizeL;
```

@miukimiu Always try to avoid pixel values when you can.